### PR TITLE
Mat 9518 update measure500 when measure id not exist

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-9518](https://jira.cms.gov/browse/MAT-9518)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/resources/MeasureController.java [hunk 1]
```diff
@@ -177,67 +177,66 @@ public ResponseEntity<Measure> updateMeasure(
...
+      throw new ResourceNotFoundException("Measure", id);
+    }
     checkMeasureLock(existingMeasure, username);
 
-    if (existingMeasure != null) {
-      if (username != null && existingMeasure.getCreatedBy() != null) {
-        log.info("got username [{}] vs createdBy: [{}]", username, existingMeasure.getCreatedBy());
-        // either owner or shared-with role
-        measureService.verifyAuthorization(username, existingMeasure);
+    if (username != null && existingMeasure.getCreatedBy() != null) {
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: Although its not part of the ticket, Im slightly confused about this condition. is this still relevant? 
1. username cant be null if it reaches here. 
2. Even if it is null we are still letting user to update the measure, which is odd
3. verifyAuthorization should be outside of if block

Comment (sb-cecilialiu) [reply src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: @adongare You are right, the condition is unnecessary, and createdBy wasn't considered for verifyAuthorization anyway, so I just removed the condition. Thanks Ashok!

Comment (adongare) [reply src/main/java/cms/gov/madie/measure/resources/MeasureController.java]: np, thanks for refactoring it.

**Review by adongare — APPROVED** [review]
